### DISCUSSION TITLE
Update of the new design in aws

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#4100] [Hubs Cloud Questions] [answered]
Hello, 
I have try to update my aws hubs cloud with the new UI design but It doesn't work . 
I have create a new folder with the hubs client, download with git clone , undeploy and deploy again 
and nothing happend, 
Have ou some information about how to deploy the new UI version in aws ? 
Thx

**Comment by johnshaughnessy** [comment, marked as answer]
It sounds like the code with the new UI isn't making it onto your cloud instance. I recommend checking that each step of the process does what you expect. e.g. Did the sign in succeed? Are you working on the latest git commit? Do you see the new UI running locally? Does the package version on your cloud instance change if you inspect it with `bio sup status` while you're `ssh`ed in and do the deploy? 

There's not enough information in your report to know which part isn't working. 

If there's information missing from this guide, let me know what isn't clear so we can improve the docs. https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

**Reply by stealink** [reply]
Hello , 

I have try to update with a clean installation , just use a domain in route 53 , I am in Ireland Zone, and the new UI isn't install . 
first test : 
1 - order my new domain 
2 - go to the website hubs and click on gets hubs cloud personnal button 
3 - go to aws and install it ( I am use to install mozilla hubs , so no worry about it ) 
4 - go to my hubs : old design appears . 

have you an idea  ? 
thx
